packages/rum-core/src/error-logging/error-logging.js

```diff
@@ -32,6 +32,44 @@ import {
 } from '../common/utils'
 import { truncateModel, ERROR_MODEL } from '../common/truncate'
 
+/**
+ * List of keys to be ignored from getting added to custom error properties
+ */
+const IGNORE_KEYS = ['stack', 'message']
+
+function getErrorProperties(error) {
+  /**
+   * Flag which is used to eliminate the empty object
+   * check on context.custom
+   */
+  let noPropertyFound = true
```

Comment:
nit: I think the using the positive here is easier to understand otherwise it would become double negative. 
e.g. `let propertyFound=false`